Add support to navigation

Lukewh opened the pull request:
## Done

Added `Support` to navigation

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check the content matches the [copy doc (second part)](https://docs.google.com/document/d/1cLgHBSlRM-yOGifOz-vexDAq8XaW-TBzmGMtJMYbRZU/edit?ts=5b473e3a#)
- Check all the links go to the right places

frankiegranato commented:
That looks excellent Luke! And so fast. 
Can you move 'Support' so it comes after 'Enterprise' please? So order would be Enterprise, Support, Developer, Community.

anasereijo commented:
Thanks @Lukewh :)

can you please make the "* Minimums apply" h6?

webteam-app commented:
Starting demo at: http://www.ubuntu.com-pr-3738.run.demo.haus/

anasereijo commented:
@Lukewh sorry forgot to mention to make "* Minimums apply" 14px

anthonydillon commented:
## QA
- on small screens you hide the prices but don't hide the note about "* Minimums apply"
- the copy doc has now links on the list items

anthonydillon commented on templates/templates/_navigation-support-h.html:
```diff
@@ -0,0 +1,184 @@
+        </div>
+      </div>
+      <div class="row">
+        <p class="p-heading--six-original">* Minimums apply</p>
```
Could you just use `<small><i>* Minimums apply<I><small>` save us another snowflake

Lukewh replied:
Done

Lukewh commented:
@anthonydillon I didn't realise that entire block was in a 'hide' so I've added an extra line to show the price on small screens. I tried wrapping the heading and the `<small>` in a `<div>` with the classes on, but it pushed the items next to each other - I've added a comment to the code.

I'm not sure what you mean - I've added the links to the items that have them in the copy doc. I've built the second version (the table one) from the doc.

webteam-app commented:
Starting demo at: http://www.ubuntu.com-pr-3738.run.demo.haus/

webteam-app commented:
Starting demo at: http://www.ubuntu.com-pr-3738.run.demo.haus/

anasereijo commented:
The navigations links on 320px screens isn't looking ok - there's not enough space for so many links.

<img width="484" alt="screen shot 2018-07-13 at 09 13 02" src="https://user-images.githubusercontent.com/36884067/42681057-8d52d2c2-867e-11e8-8be0-b02ffddc3090.png">

My suggestions is to make it possible to scroll (horizontally) like google does:

![img_8861](https://user-images.githubusercontent.com/36884067/42681511-cf6e68c8-867f-11e8-91a6-c227fb861428.PNG)

frankiegranato commented:
This is not approved by Mark. We should re-think this.

anthonydillon commented:
@Lukewh sorry, think we should kill this branch at this stage.

Lukewh commented:
@anthonydillon go ahead, no worries.
